### PR TITLE
feat: TT-223 Consume end point archive customers

### PR DESCRIPTION
**Pull request by jeffqev**

### CODE REVIEW

File: src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.spec.ts [hunk 1]
```diff
@@ -191,6 +215,60 @@ describe('CustomerTableListComponent', () => {
     expect(component.dtElement.dtInstance.then).toHaveBeenCalled();
   });
 
+  it('openModal should set on true and display "Are you sure you want to archive activity"', () => {
```
Comment (thegreatyamori) [inline src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.spec.ts:218]: Instead archive is customer?

File: src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.ts [hunk 2]
```diff
@@ -17,6 +17,7 @@ import {
 } from './../../../../store/customer-management.actions';
 import { ResetProjectToEdit, SetProjectToEdit } from '../../../projects/components/store/project.actions';
 import { ResetProjectTypeToEdit, SetProjectTypeToEdit } from '../../../projects-type/store';
+import { UnArchiveCustomer } from '../../../../store/customer-management.actions';
```
Comment (scastillo-jp) [inline src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.ts:20]: **UnArchiveCustomer**  rename to **UnarchiveCustomer**

File: src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.ts [hunk 3]
```diff
@@ -151,7 +172,20 @@ export class CustomerListComponent implements OnInit, OnDestroy, AfterViewInit {
...
+    this.message = `Are you sure you want to archive ${item.name}?`;
     this.showModal = true;
   }
+
+  switchStatus(item: CustomerUI): void {
+    if (item.key !== 'inactive') {
+      this.openModal(item);
+    } else {
+      this.showModal = false;
+      this.store.dispatch(new UnArchiveCustomer(item.id));
```
Comment (scastillo-jp) [inline src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.ts:184]: change for UnarchiveCustomer

File: src/app/modules/customer-management/store/customer-management.actions.spec.ts [hunk 4]
```diff
@@ -77,4 +77,23 @@ describe('CustomerManagmentActions', () => {
     const resetCustomerIdToEdit = new actions.ResetCustomerToEdit();
     expect(resetCustomerIdToEdit.type).toEqual(actions.CustomerManagementActionTypes.RESET_CUSTOMER_ID_TO_EDIT);
   });
+
+  it('UnArchiveCustomer type is CustomerManagementActionTypes.UNARCHIVE_CUSTOMER', () => {
```
Comment (scastillo-jp) [inline src/app/modules/customer-management/store/customer-management.actions.spec.ts:81]: change for UnarchiveCustomer

**Review by scastillo-jp — APPROVED** [review]

**Review by thegreatyamori — APPROVED** [review]

**Pull request merged**